frontend/src/pages/ForgotPassword/ForgotPassword.jsx

```diff
@@ -75,8 +86,29 @@ export function ForgotPassword(props) {
     setFormErrors(errors);
   };
 
-  function submitForgotPassword(e) {
-    return setSubmited(true);
+  function submitForgotPassword() {
+    setToast({ open: true, message: "Loading...", severity: "info" });
+    axios
+      .post(
+        `${END_POINT}/admin/forgotpassword`,
```

Comment:
[question] how are u making sure that the user is resetting his/her password only? Any identifier in the url?

Reply:
That are already done in backend side... While submitting the email in forgot password form... It will check the given email in admin collection... If not throw error User not found otherwise it send a hash token for the respective email id.